hack/install_kustomize.sh

```diff
@@ -73,7 +73,17 @@ function readlink_f {
   echo "$RESULT"
 }
 
-where="$(readlink_f $where)/"
+function find_release_url() {
+  local opsys=$1
+  local arch=$2
+
+  echo "${releases}" |\
```

Comment:
this is relying on `releases` being defined globally. Let's pass it in like you're doing for opsys and arch.